### PR TITLE
Pubs 1408

### PR DESCRIPTION
**Pull request by mbucknell**

### CODE REVIEW

**Comment by mbucknell** [comment]
I know the build will fail because I forgot to remove the mvn build portion in the .travis.yml

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+recursive-include pubs_ui/manager/static *
+recursive-include pubs_ui/pubswh/static *
+recursive-include pubs_ui/bower_components *
+recursive-include tests *
+recursive-include docs *
+recursive-include pubs_ui/auth/templates *
+recursive-include pubs_ui/manager/templates *
+recursive-include pubs_ui/pubswh/templates *
+
+recursive-exclude wqp *.pyc
```
Comment (ayan-usgs) [inline MANIFEST.in]: It looks like the `wqp` here might be a typo.

Comment (ayan-usgs) [inline MANIFEST.in]: Could probably do without it. I'm sure that wheel archives exclude .pyc files without need for it to specified; new .pyc files are created when the wheel is installed. ~~It's really more of thing to make sure any source distributions are clean.~~ Nevermind, setuptools is smart enough to not include the .pyc files in the package regardless of type.

**Comment by mbucknell** [comment]
Closing this pull request and opening a new one with effectively the same changes updated to the current code status of the project.

**Pull request closed without merging**